packages/slate-tools/tools/webpack/config/prod.js

```diff
@@ -42,6 +42,16 @@ module.exports = merge(
             },
           ],
         },
+        {
+          test: /\.css$/,
+          exclude: commonExcludes(),
+          use: extractStyles.extract({
+            fallbacl: 'style-loader',
```

Comment:
Small typo 'fallbacl' should be 'fallback'

Reply:
Thanks! Will rectify tonight as well as the build issues.